src/components/App/index.tsx

```diff
@@ -1,35 +1,43 @@
 import * as React from 'react';
 import { connect } from 'react-redux';
-import { Button, Container, Row } from 'react-bootstrap';
+import { Container, Row } from 'react-bootstrap';
+import {
```

Comment:
I don't think we really need the connected router and we still need the react-router-dom in any case, because that's the one we want to use (in combination with the connected router).